contents/docs/product-analytics/autocapture.mdx

```diff
@@ -12,29 +12,23 @@ import { ProductScreenshot } from 'components/ProductScreenshot'
 export const ImgClipboardAuocaptureLight = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/autocapture/clipboard-activity-light.png"
 export const ImgClipboardAuocaptureDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/autocapture/clipboard-activity-dark.png"
 
-> **Note:** Autocapture is available for our [JavaScript Web](/docs/libraries/js), [React](/docs/libraries/react), and [React Native](/docs/libraries/react-native) SDKs.
+PostHog can capture frontend events automatically using autocapture. This captures events like `pageview`, `screen`, `click`, `change of input`, or submission associated with a `button`, `form`, `input`, `select`, or `textarea`.
 
-PostHog can capture frontend events automatically using autocapture, such as any `click`, `change of input`, or submission associated with a `button`, `form`, `input`, `select`, or `textarea`.
+This means you don't need to manually instrument tracking for individual components, links, buttons, divs, spans, or other parts of your product. Autocapture also provides data for [heatmaps](/docs/toolbar/heatmaps).
 
-This means you don't need to manually instrument tracking for individual components, links, buttons, or other parts of your product.
-
-Autocapture is **enabled by default**. 
-
-## Disabling autocapture
-
-You can also disable autocapture in your [project settings](https://us.posthog.com/project/settings), or by setting `autocapture: false` in the [config](/docs/libraries/js#config). If _one_ of these is disabled, autocapture is disabled.
-
-> Disabling autocapture **does not disable [session recording](/docs/session-replay/manual)**. You can disable session recording by turning it off in your [project settings](https://app.posthog.com/project/settings), or using `disable_session_recording`.
+Autocapture is available for our [JavaScript Web](/docs/libraries/js), [React](/docs/libraries/react), and [React Native](/docs/libraries/react-native) SDKs and is **enabled by default**.
 
 ## Configuring autocapture
 
-> This section covers configuring autocapture in the JavaScript and React SDKs. For the React Native SDK, see our [React Native docs](/docs/libraries/react-native#autocapture).
+> This covers configuring autocapture in the JavaScript and React SDKs. For the React Native SDK, see our [React Native docs](/docs/libraries/react-native#autocapture).
+
+### Reducing events with an allow and ignorelist
 
-### Reducing events with an allowlist
+Autocapture enables you to start capturing events on your site quickly, but this can lead to large numbers of events. To counteract this, you can configure autocapture using allowlists and ignorelists. 
```

Comment:
```suggestion
Autocapture enables you to start capturing events on your site quickly, but this can lead to large numbers of events. To counteract this, you can configure autocapture using `allowlists` and `ignorelists`. 
```